pyscitt/pyscitt/cli/governance.py

```diff
@@ -31,40 +63,129 @@ def execute_governance(
             print(f"Proposal {result.id} is {result.state}")
 
 
-def propose_root_ca_certs(
-    client: Client, bundle_name: str, ca_cert_path: Path, dump_only: bool
-):
+def propose_root_ca_certs(bundle_name: str, ca_cert_path: Path):
     cert_bundle = ca_cert_path.read_text()
     proposal = governance.set_ca_bundle_proposal(bundle_name, cert_bundle)
-    execute_governance(client, proposal, dump_only)
+    return GovernanceAction(proposal)
 
 
-def propose_configuration(client: Client, configuration_path: Path, dump_only: bool):
+def propose_configuration(configuration_path: Path):
     with open(configuration_path) as f:
         configuration = json.load(f)
 
     proposal = governance.set_scitt_configuration_proposal(configuration)
-    execute_governance(client, proposal, dump_only)
+    return GovernanceAction(proposal)
 
 
-def propose_open_service(
-    client: Client, next_service_certificate_path: Path, dump_only: bool
-):
+def propose_open_service(next_service_certificate_path: Path):
     next_service_certificate = next_service_certificate_path.read_text()
     proposal = governance.transition_service_to_open_proposal(next_service_certificate)
-    execute_governance(client, proposal, dump_only)
+    return GovernanceAction(proposal)
 
 
-def propose_generic(client: Client, path: Path):
-    proposal = path.read_bytes()
-    execute_governance(client, proposal)
+def propose_generic(path: Path):
+    return GovernanceAction(path.read_bytes())
+
+
+def propose_constitution(path: Path):
+    constitution = path.read_text()
+    proposal = governance.set_constitution_proposal(constitution)
+    return GovernanceAction(proposal)
+
+
+CONSTITUTION_MARKER_START = "// ----- SCITT Constitution starts here -----\n"
```

Comment:
I think the code below is a little bit confusing semantically. It's half-way generic and half-way referring to SCITT. I think we need to decide which one it is. If it's generic, then the marker changes to something like "App Constitution Extension starts here" and there's no specific mention of scitt anywhere; maybe making the CLI command more specific is still useful like "patch_app_constitution_extension`. If the patch functionality is specific, then the variables, function names and arguments, CLI command names should probably include "scitt" more. I think I prefer the generic variant but I don't have a very strong opinion.